WooCommerce/Classes/ViewRelated/Dashboard/ProductStock/ProductStockDashboardCard.swift

```diff
@@ -285,29 +248,11 @@ private extension ProductStockDashboardCard {
             comment: "Text on the empty state of the Stock section on the My Store screen. " +
             "Reads as: No item found with Out of stock status"
         )
-        enum ContentUnavailable {
-            static let title = NSLocalizedString(
-                "productStockDashboardCard.contentUnavailable.title",
-                value: "Unable to load stock report",
-                comment: "Title when we can't load stock report because user is on a deprecated WooCommerce Version"
-            )
-            static let details = NSLocalizedString(
-                "productStockDashboardCard.contentUnavailable.details",
-                value: "Make sure you are running the latest version of WooCommerce on your site" +
-                " and enabling Analytics in WooCommerce Settings.",
-                comment: "Text that explains how to update WooCommerce to get the latest stats"
-            )
-            static let buttonTitle = NSLocalizedString(
-                "productStockDashboardCard.contentUnavailable.buttonTitle",
-                value: "Still need help? Contact us",
-                comment: "Button title to contact support to get help with deprecated stats module"
-            )
-            static let done = NSLocalizedString(
-                "productStockDashboardCard.contentUnavailable.dismissSupport",
-                value: "Done",
-                comment: "Button to dismiss the support form from the Dashboard stock card error screen."
-            )
-        }
+        static let unavailableAnalytics = NSLocalizedString(
+            "productStockDashboardCard.unavailableAnalyticsView.title",
+            value: "We can't display your store's stock analytics",
```

Comment:
Copy suggestion: 

I'd recommend removing the usage of "we" here, but instead say "Unable to display..." 

That format is more commonly used in the app if I search for the string. Also applies to other strings in the other cards.

Reply:
Good point! I updated all the wording in b71cb362fc4c8d72d031f331c56f22e310e7f631.